lib/Doctrine/Persistence/Mapping/RuntimeReflectionService.php

```diff
@@ -5,18 +5,31 @@
 namespace Doctrine\Persistence\Mapping;
 
 use Doctrine\Common\Reflection\RuntimePublicReflectionProperty;
+use Doctrine\Common\Reflection\TypedNoDefaultReflectionProperty;
 use ReflectionClass;
 use ReflectionException;
 use ReflectionMethod;
 use ReflectionProperty;
+use function array_key_exists;
 use function class_exists;
 use function class_parents;
+use function phpversion;
+use function version_compare;
 
 /**
  * PHP Runtime Reflection Service.
  */
 class RuntimeReflectionService implements ReflectionService
 {
+    /** @var bool */
+    private $supportsTypedPropertiesWorkaround;
+
+    public function __construct()
+    {
+        $this->supportsTypedPropertiesWorkaround =
```

Comment:
I don't recall us having a rule on line length, does phpcs complain if you put the code in one line? If not, I wouldn't wrap this.